docker-compose.yml

```diff
@@ -2,14 +2,14 @@ version: '2'
 services:
   nginx:
     image: nginx:1.14-alpine
-    command: nginx -g 'daemon off;' -c /etc/netbox-nginx/nginx.conf
```

Comment:
Maybe add the config directly to the config file?

Reply:
There is no actual nginx.conf to manage for us. Docker-compose is unable to start nginx with the previous instructions.
Those cli options "daemon off;" actually causes nginx not to start, because that line is in the nginx.conf. So it should just be stripped, which is my change actually.